templates/terraform/constants/domain_mapping.erb

```diff
@@ -0,0 +1,15 @@
+// While the domain mapping is setting up the certificate id, GET returns a field
+// `pendingManagedCertificateId`, which will eventually be empty when `certificateID`
```

Comment:
Isn't that field output-only? If so, what happens if you get rid of the diffsuppress (since those should only be needed for fields that are user-specified)?

Reply:
I've removed this now, however, one concern I have, and part of the reason I wanted to test it with a generated certificate, is because I think `pendingManagedCertificateId` will go from computed to empty (which is fine), but I'm guessing that `certificateId` will go from empty to computed, in which case I'm guessing we might see a diff.